ArduPlane/config.h

```diff
@@ -114,16 +114,15 @@
 #ifndef AIRSPEED_CRUISE
  # define AIRSPEED_CRUISE                12 // 12 m/s
 #endif
-#define AIRSPEED_CRUISE_CM AIRSPEED_CRUISE*100
+
 
 
 //////////////////////////////////////////////////////////////////////////////
-// MIN_GNDSPEED
+// MIN_GROUNDSPEED
 //
-#ifndef MIN_GNDSPEED
- # define MIN_GNDSPEED                   0 // m/s (0 disables)
+#ifndef MIN_GROUNDSPEED
```

Comment:
If the definitions being changed, I'd also put the MIN at the end.

Reply:
> If the definitions being changed, I'd also put the MIN at the end.

Agree